floris/tools/optimization/yaw_optimization/yaw_optimization_base.py

```diff
@@ -530,20 +530,26 @@ def _finalize(self, farm_power_opt_subset=None, yaw_angles_opt_subset=None):
         self.yaw_angles_opt = self._unreduce_variable(yaw_angles_opt_subset)
 
         # Produce output table
-        ti = np.min(self.fi.floris.flow_field.turbulence_intensity)
+        ti = np.min(self.fi.floris.flow_field.turbulence_intensities)
```

Comment:
I think the optimization classes are getting updated anyway, so this will probably get fixed elsewhere, but it doesn't seem like using the minimum TI value from turbulence_intensities is correct here.

Reply:
You're right, I think this will get reconciled either in the current #790 which is updated all optimization to 4d (@bayc)?  Or else a second follow-on pull request which will reconfigure around TimeSeries and WindRose objects

Reply:
We should create an issue for this to make sure we track it?